lambdas/data-migration2/src/storeErrors.ts

```diff
@@ -0,0 +1,58 @@
+import Logger from '@cumulus/logger';
+
+const fs = require('fs');
+const JSONStream = require('JSONStream');
+// const { finished } = require('stream');
+// const { promisify } = require('util');
+const { s3 } = require('@cumulus/aws-client/services');
+
+const logger = new Logger({ sender: '@cumulus/data-migration/storeErrors' });
+
+/**
+ * Helper to create error file write stream
+ * @param {string} migrationName         - Name of migration
+ * @param {string | undefined} timestamp - Timestamp for unit testing
+ * @returns {Object}                     - Object containing error write streams and file path
+ */
+export const createErrorFileWriteStream = (migrationName: string, timestamp?: string) => {
+  const dateString = timestamp || new Date().toISOString();
+  const filepath = `${migrationName}ErrorLog-${dateString}.json`;
+  const errorFileWriteStream = fs.createWriteStream(filepath);
+  const jsonWriteStream = JSONStream.stringify('{"errors": [\n', '\n,', '\n]}\n');
+  jsonWriteStream.pipe(errorFileWriteStream);
+
+  return { jsonWriteStream, errorFileWriteStream, filepath };
```

Comment:
We don't need to include `errorFileWriteStream` in the return here

Reply:
@markdboyd   Are you suggesting the explicit close/await on the finished stream isn't required?    Apologies if that's obtuse, I'm not sure I'm following though.